src/test/unit_tests/nc_coretest.js

```diff
@@ -299,10 +306,12 @@ async function read_account_manage(options) {
     const res = await exec_manage_cli(TYPES.ACCOUNT, ACTIONS.STATUS, cli_options);
     const json_account = JSON.parse(res);
     const account = json_account.response.reply;
-    account.access_keys[0] = {
-        access_key: new SensitiveString(account.access_keys[0].access_key),
-        secret_key: new SensitiveString(account.access_keys[0].secret_key)
-    };
+    if (account.access_keys.length > 0) {
```

Comment:
This is for handling anonymous account, I think the line:
`const cli_options = { name: get_name_by_email(options.email), show_secrets: true };`
should also handle it like you did above:
```
    const cli_options = options.name === config.ANONYMOUS_ACCOUNT_NAME ?
        {
            anonymous: true,
            uid: options.nsfs_account_config.uid,
            gid: options.nsfs_account_config.gid,
        } : {
```
(you can write it in a function and reuse it).

Reply:
As we discussed on slack, we might want to get rid of this anonymous special treatment on the CLI, so keeping it is as it is today.